cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/FoldBlocks.hs

```diff
@@ -72,5 +83,69 @@ prop_foldBlocks = H.integrationRetryWorkspace 2 "foldblocks" $ \tempAbsBasePath'
   -- tests that `foldBlocks` receives ledger state; once that happens,
   -- handler is called, which then writes to the `lock` and allows the
   -- test to finish.
-  _ <- H.evalIO $ IO.readMVar lock
+  _ <- H.evalIO $ H.timeout 30_000_000 $ IO.readMVar lock
   H.assert True
+
+prop_foldBlocks_check_if_treasury_is_growing :: H.Property
```

Comment:
why is this called `prop_foldBlocks_check_if_treasury_is_growing` if we're using `foldEpochState`? This should live in its own module.

Reply:
moved